Core/Core/Conversations/APIConversation.swift

```diff
@@ -312,7 +312,7 @@ public struct PostConversationRequest: APIRequestable {
         public let media_comment_id: String?
         public let media_comment_type: MediaCommentType?
         public let attachment_ids: [String]?
-        public let group_conversation: Bool?
+        public let group_conversation: Bool = true
         public let force_new: Bool? = nil // Setting this seems to cause the api to ignore group_conversation
```

Comment:
As per the discussion with the Android team, we must set group_conversation = true.

Reply:
Is there no use case where we might want this set to false?

Reply:
@szabinst yes, we set it true for ever.

Reply:
Even then, shouldn't this `true` be set by the UseCase? And tested there?
cc @szabinst

@Ahmed-Naguib93 Could you please provide some explanation why is this always true and comment that in code? My understanding is the following:
`group_conversation` is set to `true` in all cases, for legacy reasons. This matches Android behavior.
`bulk_messages` governs whether a message is sent individually or not

Reply:
I'd also move this to the UseCase level and add comment.